docs/index.rst

```diff
@@ -9,8 +9,53 @@ Pangeo
 .. image:: _static/pangeo_simple_logo.svg
    :width: 175 px
 
-A community platform for Big Data geoscience
-============================================
+A community promoting open, reproducible, and scalable geoscience
+=================================================================
+
+.. list-table:: 
+   :widths: 10 10 10 10 10 
+   :header-rows: 1
+   :align: left
+
+   * - ❓ Have questions about Python software?
+     - 🤩 Want to hear about new updates to Pangeo software or infrastructure?
```

Comment:
Pangeo software: what is it? Pangeo ecosystem?

Reply:
Good point! Maybe Pangeo software is poorly defined. (Though we could like to the [Pangeo Packages](https://pangeo.io/packages.html) page, which also needs a serious update!)

I like your suggestion of "Pangeo ecosystem", as it encompasses more than just the software side of Pangeo. My only hesitation there is that the term is more vague. The table I added here is largely to steer people in the right direction quickly. It's hard to know, but I'd guess that a lot of visitors to the site are there to find quick answers to coding problems, in which case explicitly stating "software" instead of "ecosystem" might be more functional. My primary goal with this table is to address some of the most frequent questions our site visitors may have. 

That all being said, happy to change to "ecosystem" if we think that's best! 😊